clumper/clump.py

```diff
@@ -62,6 +63,56 @@ def read_json(cls, path):
             return Clumper(data)
         return Clumper(json.loads(pathlib.Path(path).read_text()))
 
+    @classmethod
+    def read_csv(cls, path, delimiter=",", fieldnames=None, nrows=None):
+        """
+        Reads in a csv file. Can also read files from url.
+
+        Parameters
+        ----------
+        path : filename or url
+        delimiter: must be a single character. `,` is the default.
+        fieldnames: If `None`, the first row of the csv is assumed as the header.
+        nrows: Number of rows to read in. Useful when reading large files. If `None`, all rows are read.
+
+        Usage:
+
+        ```python
+        from clumper import Clumper
+
+        clump = Clumper.read_csv("tests/iris.csv")
```

Comment:
It might be good to show an example that also demonstrates how the `fieldnames` argument should be used.